rules-tests/Transform/Rector/MethodCall/MethodCallToMethodCallRector/Fixture/not-replacing-methode-from-current-class.php.inc

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Rector\Tests\Transform\Rector\MethodCall\MethodCallToMethodCallRector\Fixture;
+
+class AClass {
+
+    public function run()
+    {
+        $this->methodFromAClass();
+    }
+
+    public function methodFromAClass()
+    {
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Tests\Transform\Rector\MethodCall\MethodCallToMethodCallRector\Fixture;
+
+class AClass
+{
+    public function __construct(
+            private Rector\Tests\Transform\Rector\MethodCall\MethodCallToMethodCallRector\AnotherClass $anotherClass
```

Comment:
it need to have \ prefix for fqcn as inside namespace

Reply:
You're right.
It's fixed.